app/policies/user_policy.rb

```diff
@@ -13,12 +20,11 @@ def edit?
       !record.superadmin?
     when 'organisation_admin'
       allow_self_only ||
-        (record.normal? && belong_to_same_organisation_subtree?(current_user, record))
+        (record.normal? && (record.organisation && record.organisation.id == current_user.organisation.id))
```

Comment:
I don't know that `allow_organisation_admin` feels correct for the name for the whole condition, but I do like `own_organisation`.  I'll use that.

In a (unpushed) version of this change I tried to name all the conditionals, and had `allow_superadmin` or `allow_admin` etc... but in the end I found the names didn't really express the concern.  For example my `allow_admin` checked that the current user was an admin, but also that the record being acted on wasn't a superadmin and that didn't feel like the right name for that condition.  I ended up backing all that out and making just the `allow_self_only` change.  I think `own_organisation` works though.